model.json

```diff
@@ -0,0 +1,35 @@
+{
+    "participant" : {
+        "requiredAttributes" : {},
+        "plural" : "participants"
+    },
+    "sample" : {
+        "requiredAttributes" : {
+            "participant_id" : "participant"
+        },
+        "plural" : "samples"
+    },
+    "pair" : {
+        "requiredAttributes" : {
+            "case_sample_id" : "sample",
+            "control_sample_id" : "sample",
+            "participant_id" : "participant"
+        },
+        "plural" : "pairs"
+    },
+    "participant_set" : {
+        "requiredAttributes" : {},
+        "plural" : "participant_sets",
+        "memberType" : "participant"
```

Comment:
Note: The existence of the `memberType` field is the only way to determine whether an entity type is a collection type or not. I initially had an `isCollectionType` boolean field but then figured that was asking for one to be defined and not the other, so the code decides whether the type is a collection by checking for the field.